course/src/main/java/org/openedx/course/presentation/dates/CourseDatesFragment.kt

```diff
@@ -517,13 +535,24 @@ private fun CourseDateItem(
         Row(
             modifier = Modifier
                 .fillMaxWidth()
+                .padding(end = 4.dp)
                 .clickable(enabled = dateBlock.blockId.isNotEmpty() && dateBlock.learnerHasAccess,
                     onClick = { onItemClick(dateBlock.blockId) })
         ) {
+            dateBlock.dateType.drawableResId?.let { icon ->
+                Icon(
+                    modifier = Modifier
+                        .padding(end = 4.dp)
+                        .align(Alignment.CenterVertically),
+                    painter = painterResource(id = if (dateBlock.learnerHasAccess.not()) coreR.drawable.core_ic_lock else icon),
```

Comment:
not sure but I think there should be use of `VERIFIED_UPGRADE_DEADLINE` and `VERIFICATION_DEADLINE_DATE`.